client/web/src/views/components/StaticView.tsx

```diff
@@ -5,41 +5,43 @@ import { ViewProviderResult } from '@sourcegraph/shared/src/api/extension/extens
 import { TelemetryProps } from '@sourcegraph/shared/src/telemetry/telemetryService'
 import { isErrorLike } from '@sourcegraph/shared/src/util/errors'
 
-import { ViewCard } from './card/view-card/ViewCard'
-import { ViewContent } from './content/view-content/ViewContent'
-import { ViewErrorContent } from './content/view-error-content/ViewErrorContent'
-import { ViewLoadingContent } from './content/view-loading-content/ViewLoadingContent'
+import * as View from './view'
 
 interface StaticView extends TelemetryProps, React.DetailedHTMLProps<React.HTMLAttributes<HTMLElement>, HTMLElement> {
-    view: ViewProviderResult
+    content: ViewProviderResult
 }
 
 /**
  * Component that renders insight-like extension card. Used by extension views in extension
  * consumers that have insight section (the search and the directory page).
  */
 export const StaticView: React.FunctionComponent<StaticView> = props => {
-    const { view, telemetryService, ...otherProps } = props
+    const { content, telemetryService, ...otherProps } = props
+    const view = content.view
```

Comment:
Does it make sense to destructure all props consistently here?

```suggestion
    const { content: { view, id: contentId }, telemetryService, ...otherProps } = props
```